src/media/css/feed/feed.styl

```diff
@@ -182,6 +183,13 @@
             text-overflow: ellipsis;
             white-space: nowrap;
         }
+        .desc {
+            font-size: 16px;
```

Comment:
it would be wrong according to

![](http://i.imgur.com/kwb6056.png)

granted maybe we should update that mixin